### ISSUE TITLE
Binding methods of the super class

### ISSUE THREAD

**Issue by DaedricSpartan99** [godotengine/godot-cpp#389]
The compiler seems not accepting this:
```
class A {
    // various fields
public:
    void example();
};

class B : public Node, private A {
    GODOT_CLASS(B, Node)
public:
    // .... basic class initialization
    
    // setting external visibility
    using A::example;
    // method registering 
    static void _register_methods() {
        register_method("example", &B::example);
    }
};
```
Is there an elegant way to bind the A class `example` method without re-implementing it into
the B class with a different name?

**Comment by Zylann** [comment]
What does the compiler say? I'm not sure if that's related to C++ bindings.

**Comment by Zylann** [comment]
Looks like this would be fixed by https://github.com/godotengine/godot-cpp/pull/304

**Comment by Zylann** [comment]
Fixed by #304
